tests/fast-integration/installer/index.js

```diff
@@ -175,7 +174,8 @@ async function chartList(options) {
   const isGardener = process.env["GARDENER"] || (gardernerDomain) ? "true" : "false";
   const domain = process.env["KYMA_DOMAIN"] || gardernerDomain || "local.kyma.dev";
   const isBEBEnabled = !!options.newEventing;
-  const overrides = `global.isLocalEnv=false,global.ingress.domainName=${domain},global.environment.gardener=${isGardener},global.domainName=${domain},global.tlsCrt=ZHVtbXkK,global.isBEBEnabled=${isBEBEnabled}`;
+  const isCompassEnabled = !!options.withCompass;
+  const overrides = `global.isLocalEnv=false,global.ingress.domainName=${domain},global.environment.gardener=${isGardener},global.domainName=${domain},global.tlsCrt=ZHVtbXkK,global.isBEBEnabled=${isBEBEnabled}${isCompassEnabled ? ",global.disableLegacyConnectivity=true": ""}`;
```

Comment:
I would not use a ternary operator inside an expression. I would just add the true/false flag:
```suggestion
  const overrides = `global.isLocalEnv=false,global.ingress.domainName=${domain},global.environment.gardener=${isGardener},global.domainName=${domain},global.tlsCrt=ZHVtbXkK,global.isBEBEnabled=${isBEBEnabled},global.disableLegacyConnectivity=${isCompassEnabled}`;
```